app/Run.hs

```diff
@@ -178,7 +179,7 @@ runNode nodeCli@NodeCLIArguments{..} loggingLayer cc = do
 -- | Sets up a simple node, which will run the chain sync protocol and block
 -- fetch protocol, and, if core, will also look at the mempool when trying to
 -- create a new block.
-handleSimpleNode :: forall blk. (RunNode blk, TraceConstraints blk)
+handleSimpleNode :: forall blk. (RunNode blk, TraceConstraints blk, Show (GenTxId blk))
```

Comment:
There is no need for this change as `Show (GenTxId blk)` is part of `TraceConstraints`.

Reply:
Ah yes, it is now after we rebased on the other PR. Thanks.